lib/brains/forms.py

```diff
@@ -83,5 +83,10 @@ def braintree_data(self):
             'payment_method_token': self.cleaned_data['paymethod'].provider_id,
             'plan_id': self.seller_product.external_id,
             'trial_period': False,
-            'name': 'Mozilla',
+            'descriptor': {
+                # TODO: figure out how to get product in here
+                # https://github.com/mozilla/payments/issues/57
+                'name': 'Mozilla *product',
```

Comment:
I tested this and oddly it cannot have a space in it. It must be ``Mozilla*product`` where the final string is less than 22 characters. Maybe until we fix #57 we could make it ``Mozilla*subscription`` but whatever, just a placeholder.